app/models/git_hub_api/issue.rb

```diff
@@ -67,5 +71,21 @@ def applied_labels
     def update(options)
       GitHubApi.execute(@client, :update_issue, @repo_name, @number, @title, @body, options)
     end
+
+    def update_title(new_title)
+      GitHubApi.execute(@client, :update_issue, @repo_name, @number, :title => new_title)
+    end
+
+    def wipify_title
+      if @title !~ WIP_REGEX
+        update_title("[WIP] #{@title}")
+      end
```

Comment:
This will not catch instances where `[WIP]` is not at the front, such as `[DARGA] [WIP] ...`. For this reason I have the .pr_title_tags on the Branch model which parses it into an array.

Even so, I may merge without this fix in place, and then we can do that in a follow up.